### PR TITLE
refactor(ci): `workflow_dispatch` for release wflow

### PR DESCRIPTION
**Pull request by Yohe-Am**
- Manual trigger for release wflow
- Adds missing ghjk step for `vscode-extension` release job

#### Motivation and context

Provides recovery path for when things break.

#### Migration notes

Doesn't affect end users.

### Checklist

- [ ] The change come with new or modified tests
- [ ] Hard-to-understand functions have explanatory comments
- [ ] End-user documentation is updated to reflect the change

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -11,36 +23,50 @@ env:
 jobs:
   check-bump:
     runs-on: ubuntu-latest
-    if: github.ref_type == 'tag'
-
+    if: github.ref_type == 'tag' 
+      || (
+        github.event_name == 'workflow_dispatch'
+        && inputs.checkBump
```
Comment (afmika) [inline .github/workflows/release.yml:29]: Question, how is this event triggered?

**Review by afmika — DISMISSED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (Yohe-Am) [reply .github/workflows/release.yml:29]: Workflow dispatch can be done through the github ui, in the actions tab, or their REST API.

**Comment by Yohe-Am** [comment]
I thought these flakes we're seeing when deno downloads the modules was a problem with `deno.land/x` but it just flaked out with an npm package. It just might be bug in the deno resolver. Let's be upgrade to latest deno (1.39.2) and ig I'll go looking for a ticket.
image

**Comment by Yohe-Am** [comment]
Strange thing is I've never seen this locally, only on the CI and I've cleaned out my DENO_DIR scores of times over the past week.

**Review by afmika — APPROVED** [review]

**Comment by Yohe-Am** [comment]
Regarding the bug Natondoro run into [here](https://github.com/denoland/deno/issues/21866), looks like that was indeed a registry issue as discussed [here](https://github.com/denoland/deno/issues/21866).

**Pull request merged**